base_layer/wallet_ffi/.github/workflows/release.yml

```
@@ -0,0 +1,171 @@
+        maturin build --release --features python-bindings --out dist
+        
+    - name: Upload wheels
+      uses: actions/upload-artifact@v3
+      with:
+        name: wheels-${{ matrix.os }}
+        path: base_layer/wallet_ffi/dist/*.whl
+
+  build-sdist:
+    name: Build source distribution
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)